src/main/resources/swagger/api-docs.yaml

```diff
@@ -4080,6 +4080,81 @@ paths:
           - email
           - profile
 
+  /api/workspaces/{workspaceNamespace}/{workspaceName}/snapshots:
+    post:
+      x-passthrough: true
+      x-passthrough-target: rawls
+      responses:
+        201:
+          description: Created
+          schema:
+            $ref: '#/definitions/WMDataReferenceResponse'
+        404:
+          description: Workspace or snapshot not found
+      parameters:
+        - in: path
+          description: workspace namespace
+          name: workspaceNamespace
+          required: true
+          type: string
+        - in: path
+          description: workspace name
+          name: workspaceName
+          required: true
+          type: string
```

Comment:
namespace and name can use:
```
      parameters:
        - $ref: '#/parameters/workspaceNamespaceParam'
        - $ref: '#/parameters/workspaceNameParam'
```
to centralize and cut down on lines of code in the file